pkg/limatmpl/locator.go

```diff
@@ -312,6 +417,15 @@ func TransformCustomURL(ctx context.Context, locator string) (string, error) {
 		return newLocator, nil
 	}
 
+	if u.Scheme == "github" {
+		newLocator, err := transformGitHubURL(ctx, u.Opaque)
+		if err != nil {
+			return "", err
+		}
+		logrus.Debugf("GitHub locator %q replaced with %q", locator, newLocator)
+		return newLocator, nil
+	}
+
 	plugin, err := plugins.Find("url-" + u.Scheme)
```

Comment:
Any reason not to implement the github locator as a plugin?

Reply:
I had implemented it as a plugin in bash first, but that is not portable to Windows.

I've been thinking of `github:` as been an "internal" scheme, like `template:`, meaning you cannot modify it via plugin, but it will have the same semantics everywhere.